source/sync/migrate-sync-modes.txt

```diff
@@ -168,7 +174,7 @@ a client reset. Then, they can communicate with your App's Flexible Sync backend
 You can disable and re-enable Flexible Sync and client apps will still
 communicate with your backend.
 
-If you re-enable Partitition-Based Sync you will lose the "migrated" status.
```

Comment:
lol. "Partitition" is a fun typo. "Partytition-Based Sync" will be the next Sync Type. Just you wait. 🥳

Reply:
haha! this one's new. I've stumbled on plenty of "Partion"s though